OpenSim/Simulation/Model/Frame.h

```diff
@@ -193,6 +206,41 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
     */
     SimTK::Vec3 findLocationInAnotherFrame(const SimTK::State& state, 
                     const SimTK::Vec3& point_F, const Frame& otherFrame) const;
+
+    /**
+    Take a point located and expressed in this frame (F) and determine
+    its location expressed in Ground (G). This method is equivalent to
+    `findLocationInAnotherFrame()` where `otherFrame` is always Ground.
+
+    @param state       The state of the model.
+    @param point_F     The point to be re-expressed.
+    @return point_G    The re-expression of the point in Ground.
+    */
+    SimTK::Vec3 findLocationInGround(const SimTK::State& state,
+                    const SimTK::Vec3& point_F) const;
+
+    /**
+    Take a point located and expressed in this frame (F) and determine
+    its velocity expressed in Ground (G).
+
+    @param state       The state of the model.
+    @param point_F     The point to be re-expressed.
+    @return point_G    The re-expression of the point in Ground.
+    */
+    SimTK::Vec3 findVelocityInGround(const SimTK::State& state,
```

Comment:
Hm, there may be a logical conflict with these names. I see the preference for "velocity" when referring to linear velocity, but when written alongside the other two methods, it seems like the methods with the "spatial" and "angular" adjectives would return subsets of whatever is returned by the method with no adjective. Also, someone who generates formulations using spatial vectors may refer to the Vec6 as simply "the velocity". Perhaps `Vec6 getVelocityInGround()`, `Vec3 getAngularVelocityInGround()`, and `Vec3 getLinearVelocityInGround()` would be preferable?